.github/workflows/jit.yml

```diff
@@ -26,8 +26,20 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  interpreter:
+    name: Interpreter (Debug)
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    steps:
+      - uses: actions/checkout@v4
+      - name: If this fails, it's not the JIT's fault!
+        run: |
+          ./configure --enable-experimental-jit=interpreter --with-pydebug
```

Comment:
Do we want to split the build and test steps out here? The additional granularity might be nice

Reply:
Good idea.